src/components/Intro.vue

```diff
@@ -47,6 +58,9 @@ export default {
     player() {
       return this.$refs.youtube.player;
     },
+    ismobile(){
+      return this.$mq === "xs" || this.$mq === "sm";
```

Comment:
There is actually a function, **isMinimal** in js/utils.js
You can use that here.

Reply:
Done